docs/_templates/include/schedule2021.rst

```diff
@@ -1,3 +1,5 @@
+.. while called schedule2021, this file applies to 2021 *and newer*
```

Comment:
What were the people who designed this hack thinking :japanese_ogre: shall we fix?